### PR TITLE
Ekr wglc comments

### PR DESCRIPTION
**Pull request by ekr**
Hopefully some minor editorial changes.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -887,12 +890,12 @@ matching stream and ignore further data arriving on that stream.  Without the
...
-receiver definitively knows how many bytes were sent on that stream before the
-RESET_STREAM frame, and the receiver MUST use the final size of the stream to
-account for all bytes sent on the stream in its connection level flow
-controller.
+To remedy this issue, a RESET_STREAM frame ({{frame-reset-stream}})
+includes the final size of data sent on the stream.  On receiving a
+RESET_STREAM frame, a receiver knows how many bytes the sender claims
+were sent on that stream before the RESET_STREAM frame, and the
+receiver MUST use the final size of the stream to account for all
+bytes sent on the stream in its connection level flow controller.
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:898]: I think that this is just removing a single word, but the reflow makes it hard to parse.

C-u 8 0 M-x set-fill-column

Or install editorconfig.

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -1297,10 +1300,10 @@ When implementing the server role, applications need to be able to:
...
 - listen for incoming connections, which prepares for the exchange described in
   {{handshake}};
-- if Early Data is supported, embed application-controlled data in the TLS
+- if 0-RTT data is supported, embed application-controlled data in the TLS
   resumption ticket sent to the client; and
-- if Early Data is supported, retrieve application-controlled data from the
-  client's resumption ticket and enable rejecting Early Data based on that
+- if 0-RTT data is supported, retrieve application-controlled data from the
+  client's resumption ticket and enable rejecting 0-RTT data based on that
   information.
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:1307]: I think that we need to get @MikeBishop to fight this out.

File: draft-ietf-quic-transport.md [hunk 3]
```diff
@@ -1527,12 +1530,13 @@ Connection ID is chosen by the recipient of the packet and is used to provide
...
-establish the connection IDs in each direction. Each endpoint uses the Source
-Connection ID field to specify the connection ID that is used in the Destination
-Connection ID field of packets being sent to them. Upon receiving a packet, each
-endpoint sets the Destination Connection ID it sends to match the value of the
-Source Connection ID that it receives.
+During the handshake, packets with the long header ({{long-header}})
+are used to establish the connection IDs in each direction. Each
+endpoint uses the Source Connection ID field to specify the connection
+ID that is used in the Destination Connection ID field of packets
+being sent to them. Upon receiving the first packet from the peer,
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:1537]: Again, this seems to be just one change.

Should this be "first Initial" instead.

File: draft-ietf-quic-transport.md [hunk 4]
```diff
@@ -2508,11 +2514,11 @@ transport parameter in the TLS handshake.
 Servers MAY communicate a preferred address of each address family (IPv4 and
 IPv6) to allow clients to pick the one most suited to their network attachment.
 
-Once the handshake is confirmed, the client SHOULD select one of the two
-server's preferred addresses and initiate path validation (see
+Once the handshake is confirmed, the client SHOULD select one of the two values
+preferred addresses provided by the server and initiate path validation (see
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:2518]: Redundancy here:

```suggestion
Once the handshake is confirmed, the client SHOULD select one of the two
preferred addresses provided by the server and initiate path validation (see
```

File: draft-ietf-quic-transport.md [hunk 5]
```diff
@@ -2666,7 +2672,9 @@ source address.
 
 If a max_idle_timeout is specified by either peer in its transport parameters
 ({{transport-parameter-definitions}}), the connection is silently closed
-and its state is discarded when it remains idle for longer than the minimum of
+and its state is discarded when it remains idle for longer than either
+
+the minimum of
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:2677]: See #3847.

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (ekr) [reply draft-ietf-quic-transport.md:898]: Yes, that's the change. Of course, you could also have decided to use a fill column that emacs already had :)

File: draft-ietf-quic-transport.md (hunk 2 above)
Comment (ekr) [reply draft-ietf-quic-transport.md:1307]: I'm fine with Early Data but it was undefined, so this seemed like the easiest fix. But a defn. would work too.

File: draft-ietf-quic-transport.md (hunk 3 above)
Comment (ekr) [reply draft-ietf-quic-transport.md:1537]: yes, I think first Initial

File: draft-ietf-quic-transport.md (hunk 4 above)
Comment (ekr) [reply draft-ietf-quic-transport.md:2518]: This is what the text already said, but I think its confusing. My suggestion is "values for preferred address provided by the server

File: draft-ietf-quic-transport.md (hunk 2 above)
Comment (MikeBishop) [reply draft-ietf-quic-transport.md:1307]: It doesn't help that RFC 8446 uses "0-RTT Data" and "Early Data" almost equal numbers of times.  We should have picked one and stuck to it.  Ah well -- we can do the same thing RFC 8446 does and put ("Early Data") as a synonym after the first mention of 0-RTT.

Comment (ekr) [reply draft-ietf-quic-transport.md:1307]: > It doesn't help that RFC 8446 uses "0-RTT Data" and "Early Data" almost equal numbers of times.

Happy to be of service

> Ah well -- we can do the same thing RFC 8446 does and put ("Early Data") as a synonym after the first mention of 0-RTT.

That would be fine. I'm really fine with any resolution here that is clear.
